bitbots_quintic_walk/test/rostests/test_walk.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+import time
+
+import rospy
+from bitbots_msgs.msg import JointCommand, SupportState
+from geometry_msgs.msg import Twist, Point, Pose, Quaternion
+
+from bitbots_test.mocks import MockSubscriber
+from bitbots_test.test_case import WebotsTestCase
+from nav_msgs.msg import Odometry
+
+
+class TestWalk(WebotsTestCase):
+
+    def test_start(self):
+        """ test if node starts correctly without warnings/errors/criticals
+        and if it is still there after some time (did not crash by itself)"""
+        # wait to make sure node is up
+        sub = MockSubscriber("DynamixelController/command", JointCommand, tcp_nodelay=True)
+        sub.wait_until_connected()
```

Comment:
You could also use [`assertNodeRunning`](https://github.com/bit-bots/bitbots_tools/blob/6127589573e4d223ee315608b14383ee89029777/bitbots_test/src/bitbots_test/test_case.py#L167) which might be better to read